pkg/chrootuser/user.go

```diff
@@ -18,15 +18,19 @@ var (
 // it will use the /etc/passwd and /etc/group files inside of the rootdir
 // to return this information.
 // userspec format [user | user:group | uid | uid:gid | user:gid | uid:group ]
-func GetUser(rootdir, userspec string) (uint32, uint32, error) {
+func GetUser(rootdir, userspec string) (uint32, uint32, string, error) {
 	var gid64 uint64
 	var gerr error = user.UnknownGroupError("error looking up group")
-
+	usr, uerr := user.Current()
```

Comment:
This is the user who's running buildah, and not the value that might be set in the `/etc/passwd` in the rootfs, which is the value we want to read, and default to `/` if we can't find anything in there.

Reply:
Thanks @nalind.  If you've a quick easy way to do that, please let me know, otherwise I'll go gronking.  If there's nothing found in the rootfs /etc/passwd too, is this a reasonable last chance value to add?

Reply:
You'll want to look at `parseNextPasswd()` in `chrootuser/user_unix.go`, and bubble the value it returns (and that its callers return) up to where it's needed.  The last-ditch default that `runc` uses (per [`setupUser`](https://github.com/opencontainers/runc/blob/b9b6cc6e47fe4f2aa4f744a1fc62d248c182d28d/libcontainer/init_linux.go#L240)) is "/".

Reply:
My patch below allows you to get the homedir from the /etc/passwd in the container based on the UID.  Basically what @nalind is suggesting.